docs/examples/general/reinterpret.ipynb

```diff
@@ -0,0 +1,637 @@
+{
```

Comment:
<p>as if it has -&gt; as it if had</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2791/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>